guestbook/comments/forms.py

```diff
@@ -0,0 +1,19 @@
+from django import forms
+
+from .models import Comment, Message
+
+class CreateCommentForm(forms.ModelForm):
+    text = forms.CharField(
+        label="Comment",
+        max_length=1024,
+        widget=forms.Textarea)
```

Comment:
From what I can see you're trying to override the widget? Why not use the `widgets` attribute on the `Meta` class as shown in https://docs.djangoproject.com/en/1.8/topics/forms/modelforms/#overriding-the-default-fields ?